lightspeed/modules/proc_configure-vscode-extension.adoc

```diff
@@ -19,7 +19,7 @@ You can configure the Ansible VS Code extension to enable Ansible Lightspeed, an
 . Select *Ansible Lightspeed* settings, and specify the following information:
 .. Select the *Enable Ansible Lightspeed* check box.
```

Comment:
Flagged by Vale: Use 'checkbox' rather than 'check box'.Vale[RedHat.TermsErrors](https://redhat-documentation.github.io/vale-at-red-hat/docs/main/reference-guide/termserrors/)

Reply:
Updated.